src/client/components/PhoneNumber/PhoneNumber.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react'
 import { format as formatDate, getDay } from 'date-fns'
```

Comment:
Now I start to thinking this component is too specific to call center phone numbers to be called just `PhoneNumber`. Maybe rename it to something like `CallcenterPhoneNumber`? What do you think?

Reply:
Tried to keep changes minimal but I think you are right. I will fix